client/rpc/src/state/tests.rs

```diff
@@ -177,7 +177,7 @@ fn should_send_initial_storage_changes_and_notifications() {
 
 	{
 		let mut client = Arc::new(substrate_test_runtime_client::new());
-		let api = new_full(client.clone(), Subscriptions::new(Arc::new(remote)));
+		let api = new_full(client.clone(), Subscriptions::new(Arc::new(remote)), DenyUnsafe::No);
```

Comment:
I would suggest add method that creates with `DenyUnsafe::No` instead of passing it every time

Reply:
Added a `new_full_test` macro not to repeat numerous bounds from `new_full` again here